_rules/role-attribute-has-valid-value-674b10.md

```diff
@@ -40,7 +40,7 @@ Older browsers do not support more than one token in the value for a role attrib
 ## Background
 
 - [Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
-- [WAI-ARIA 1.1 Categorization of Roles](https://www.w3.org/TR/wai-aria-1.1/#roles_categorization)
+- [WAI-ARIA 1.1 Categorisation of Roles](https://www.w3.org/TR/wai-aria-1.1/#roles_categorization)
```

Comment:
That document seems to be written in American English, and the name of that Section is "Categori**Z**ation of Roles". I think it is improper to cite it differently.